binchihua/src/common/hashmap/hashmap.c

```diff
@@ -2,13 +2,15 @@
 // Use of this source code is governed by an MIT-style
 // license that can be found in the LICENSE file.
 
-#include "hashmap.h"
```

Comment:
This is the hashmap library Artiom copied from [here](https://github.com/tidwall/hashmap.c) (I think). Should we really take care of compiler warnings from an imported library?

Reply:
Once code has been brought into the project, we're responsible for it.  The compiler can be used to help us evaluate the quality of the code that is brought in.  We're shooting for a high bar for the quality of the software (FuSa), so getting rid of warnings in imported code is worthwhile for that reason.  Fixing the warnings also prompts us to take a closer look at the code in question to see what's being brought in.